main.go

```diff
@@ -64,14 +73,35 @@ func main() {
 
 func (opts Options) update() {
 
-	oldVersionTmp := manipulator.GetVersion(opts.Filepath)
-	oldVersion := "'" + oldVersionTmp + "'"
-	fmt.Println("Application Version:", oldVersion)
+	oldVersionTmp := ""
+	if opts.Filepath != "" {
+		oldVersionTmp = manipulator.GetVersion(opts.Filepath)
+	} else if opts.Version != "" {
+		oldVersionTmp = opts.Version
+		if oldVersionTmp != "" {
+			if strings.Contains(oldVersionTmp, "v") {
+				oldVersionTmp = strings.ReplaceAll(oldVersionTmp, "v", "")
+			} else if strings.Contains(oldVersionTmp, "V") {
+				oldVersionTmp = strings.ReplaceAll(oldVersionTmp, "V", "")
+			}
+		}
+	} else {
+		fmt.Println(help.Default())
+		os.Exit(1)
+	}
 
+	oldVersion := "'" + oldVersionTmp + "'"
 	newVersion := manipulator.IncrementVersion(oldVersion, opts.Type)
-	fmt.Println("New Version:", newVersion)
 
-	manipulator.WriteVersionOnFile(opts.Filepath, oldVersion, newVersion)
+	if opts.Filepath != "" {
+		manipulator.WriteVersionOnFile(opts.Filepath, oldVersion, newVersion)
+		fmt.Println("Application Version:", oldVersion)
+		fmt.Println("New Version:", newVersion)
+	} else if opts.Version != "" {
+		//os.Setenv("newVersionApp", newVersion)
+		//fmt.Println("newVersionApp:", os.Getenv("newVersionApp"))
```

Comment:
These comments can be removed, to make the code more readable.

Reply:
Great. I am going to do that!!